src/gc/softwarewritewatch.h

```diff
@@ -204,9 +199,13 @@ inline void SoftwareWriteWatch::EnableForGCHeap()
 
     VerifyCreated();
     assert(!IsEnabledForGCHeap());
+    g_gc_sw_ww_enabled_for_gc_heap = true;
 
-    g_sw_ww_enabled_for_gc_heap = true;
-    SwitchToWriteWatchBarrier(true);
+    WriteBarrierParameters args = {};
```

Comment:
is this standard C++? MSVC rejects it and GCC issues a warning in pedantic mode that C99 style initializers like this aren't permitted in C++.